circle.yml

```diff
@@ -5,7 +8,7 @@ deployment:
       - ./build-docs.sh
       - git push git@heroku.com:openmctweb-demo.git $CIRCLE_SHA1:refs/heads/master
   openmctweb-staging-un:
-    branch: search
+    branch: open199
```

Comment:
This is temporary to allow testing on the staging server and bypass of tests. Will be removed before production.